popup.html

```diff
@@ -5,7 +5,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>minido - productivity tool</title>
     <link rel="stylesheet" type="text/css"href="popup.css" />
-    <!-- Import jQuery, Google Icons and Open Sans Font -->
```

Comment:
ew jquery, remove that

Reply:
yeah i know its shitty but it gets the work done and dead line doesnt allow us to refactor right now